airflow/models/taskinstance.py

```diff
@@ -1482,6 +1482,9 @@ def _execute_task(self, context, task_copy):
             # this task was scheduled specifically to fail.
             if self.next_method == "__fail__":
                 next_kwargs = self.next_kwargs or {}
+                traceback = self.next_kwargs.get("traceback")
+                if traceback is not None:
+                    self.log.error("Trigger failed:\n%s", "\n".join(traceback))
```

Comment:
```suggestion
                    self.log.error("Trigger failed:\n%s", "".join(traceback))
```
when i tested this locally, the lines already were newline-terminated